libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -158,6 +181,59 @@ export class DropdownComponent implements AfterContentChecked, OnDestroy, Contro
     }
   }
 
+  ngAfterViewInit() {
+    this.initializeAlignment();
+  }
+
+  private initializeAlignment() {
+    if (!this.intersectionObserverRef) {
+      const options = {
+        rootMargin: '0px',
+      };
+      const callback: IntersectionObserverCallback = (entries) => {
+        // Only apply alignment when opening:
+        if (!this._isOpening) {
+          return;
+        }
+        const entry = entries[0];
+        const isVisible = entry.boundingClientRect.width > 0;
+        if (isVisible && entry.intersectionRatio < 1) {
+          // entry not fully showing:
+          // console.log('entry not fully showing:', entry);
```

Comment:
Are all the console logs kept for future use? Consider deleting them if not :-)

Reply:
:fire: Removed :)